docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -74,7 +74,7 @@ From there, the data can be published to external platforms like Elasticsearch,
 === Does {cdc_cass} support Kubernetes?
 
 Yes.
-You can run the {csc_pulsar} on Luna Streaming or Apache Pulsar running on any Kubernetes platform, including Minikube, K8d, Kind, Google Kubernetes Engine (GKE), Microsoft Azure Kubernetes Service, Amazon Kubernetes Service (AKS), and other commonly used platforms.
+You can run the {cdc_pulsar} on Luna Streaming or Apache Pulsar running on any Kubernetes platform, including Minikube, K8d, Kind, Google Kubernetes Engine (GKE), Microsoft Azure Kubernetes Service, Amazon Kubernetes Service (AKS), and other commonly used platforms.
 You can deploy the change agent with Cassandra on Kubernetes with the https://github.com/datastax/cass-operator[cass-operator].
```

Comment:
Are we sure about this? Unless this has been changed very recently, I don't think this is true.